content/source/docs/cloud/api/feature-sets.html.md

```diff
@@ -0,0 +1,233 @@
+        "identifier": "team",
+        "is-current": true,
+        "is-free-tier": false,
+        "name": "Team",
+        "plan": "plan_G13GVoKwS5xDEW",
+        "self-serve-billing": true,
+        "cost-estimation": false,
+        "sentinel": false,
+        "teams": true,
+        "user-limit": null,
+        "audit-logging": false,
+        "sso": false,
+        "private-networking": false
+      }
+    },
+    {
+      "id": "fs-EP5Niczf2xKUFDXt",
+      "type": "feature-sets",
+      "attributes": {
+        "comparison-description": "Policy management and cost insights for runs and workspaces.",
+        "cost": 7000,
+        "description": "Team plan features plus Sentinel policy as code framework",
+        "identifier": "governance",
+        "is-current": true,
+        "is-free-tier": false,
+        "name": "Team & Governance",
+        "plan": "plan_G13T64ifEk3z92",
+        "self-serve-billing": true,
+        "cost-estimation": true,
+        "sentinel": true,
+        "teams": true,
+        "user-limit": null,
+        "audit-logging": false,
+        "sso": false,
+        "private-networking": false
+      }
+    },
+    {
+      "id": "fs-T9BCyZi3KJyWHebk",
+      "type": "feature-sets",
+      "attributes": {
+        "comparison-description": "Try out the Team & Governance plan features for 30 days. No credit card required.",
+        "cost": 0,
+        "description": "Try out the Team & Governance plan features for 30 days",
+        "identifier": "trial",
+        "is-current": true,
+        "is-free-tier": true,
+        "name": "Trial",
+        "plan": null,
+        "self-serve-billing": true,
+        "cost-estimation": true,
+        "sentinel": true,
+        "teams": true,
+        "user-limit": null,
+        "audit-logging": false,
+        "sso": false,
+        "private-networking": false
+      }
+    },
+    {
+      "id": "fs-EvCGYfpx9CVRzteA",
+      "type": "feature-sets",
+      "attributes": {
+        "comparison-description": "Essential collaboration features for practitioners and small teams.",
+        "cost": 0,
+        "description": "State storage, locking, run history, VCS integration, private module registry, and remote operations",
+        "identifier": "free",
+        "is-current": true,
+        "is-free-tier": true,
+        "name": "Free",
+        "plan": null,
+        "self-serve-billing": true,
+        "cost-estimation": false,
+        "sentinel": false,
+        "teams": false,
+        "user-limit": 5.0,
+        "audit-logging": false,
+        "sso": false,
+        "private-networking": false
+      }
+    }
+  ]
+}
+```
+
+## List Feature Sets for Organization
+
+This endpoint lists the feature sets available to a particular organization. The results may differ from the previous global endpoint - for instance, if the organization has already had a free trial, the trial feature set will not appear in this list.
```

Comment:
Question: How's this interact with the Business tier? My expectation would be that it'd appear in the global list but NOT the org list (usually), because you can't just swipe a credit card to get it... but I'm not seeing it in the global list either.

Reply:
This is a weird thing that I wasn't 100% sure how best to represent here! The Business tier should  appear in both, so I must have copy-pasted something wrong at some point, but as you note, organizations can't actually subscribe to it directly.

Maybe better phrasing would be:

```suggestion
This endpoint lists the feature sets a particular organization is eligible to access. The results may differ from the previous global endpoint - for instance, if the organization has already had a free trial, the trial feature set will not appear in this list.
```